tool/src/kotlin/mod.rs

```diff
@@ -2514,4 +2527,56 @@ mod test {
             ty_gen_cx.gen_trait_def(trait_def, &trait_name, "dev.gigapixel", "somelib");
         insta::assert_snapshot!(result)
     }
+
+    #[test]
+    fn test_opaque_gen_with_mocking_interface() {
+        let tk_stream = quote! {
+            #[diplomat::bridge]
+            mod ffi {
+                #[diplomat::opaque]
+                #[diplomat::attr(kotlin_test, generate_mocking_interface)]
+                struct MyOpaqueStruct<'b> {
+                    a: SomeExternalType
+                }
+
+                impl<'b> MyOpaqueStruct<'b> {
+
+                    pub fn get_byte() -> u8 {
+                        unimplemented!()
+                    }
+
+                    pub fn get_string_wrapper(in1: i32) -> i32 {
+                        unimplemented!()
+                    }
+                }
+
+            }
+        };
+        let tcx = new_tcx(tk_stream);
+        let mut all_types = tcx.all_types();
+        if let (_id, TypeDef::Opaque(opaque_def)) = all_types
+            .next()
+            .expect("Failed to generate first opaque def")
+        {
+            let eror_store = ErrorStore::default();
+            let docs_urls = HashMap::new();
+            let docs_generator =
+                diplomat_core::hir::DocsUrlGenerator::with_base_urls(None, docs_urls);
+            let formatter = KotlinFormatter::new(&tcx, None, &docs_generator);
+            let mut callback_params = Vec::new();
+            let mut ty_gen_cx = TyGenContext {
+                tcx: &tcx,
+                formatter: &formatter,
+                result_types: RefCell::new(BTreeSet::new()),
+                option_types: RefCell::new(BTreeSet::new()),
+                errors: &eror_store,
+                callback_params: &mut callback_params,
+            };
+            let type_name = opaque_def.name.to_string();
+            // test that we can render and that it doesn't panic
+            let (_, result) =
+                ty_gen_cx.gen_opaque_def(opaque_def, &type_name, "dev.gigapixel", "somelib", true);
```

Comment:
This is definitely my fault for introducing it in the first place 😅 but could you get rid of my personal handle?
```suggestion
                ty_gen_cx.gen_opaque_def(opaque_def, &type_name, "dev.diplomattest", "somelib", true);
```